src/plugins/eqht-css/_base.scss

```diff
@@ -0,0 +1,17 @@
+.wb-eqht-grd{
+	align-items: stretch;
+	display: flex;
+	flex-wrap: wrap;
+	height: 100%;
+
+	>[class*="col-"]{
+		display:flex;
+		flex-direction: column;
+		flex-grow:1;
+	}
+}
+.wb-eqht-grd .hght-inhrt{
```

Comment:
Merge this block with the previous one.

And then add this selector: >[class*="col-"]>section